packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -70,12 +68,36 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
     [estimatedGas, gasPrice]
   )
 
-  const approvalFeeText = useMemo(() => {
+  const ethFeeText = useMemo(() => {
     const eth = formatAmount(estimatedGasFees, { symbol: ether.symbol })
+    return eth
+  }, [estimatedGasFees])
+
+  const usdFeeText = useMemo(() => {
     const usd = formatUSD(ethToUSD(estimatedGasFees))
-    return `${eth}${isEthereumMainnet ? ` (${usd})` : ''}`
+    return `${isEthereumMainnet ? ` (${usd})` : ''}`
   }, [estimatedGasFees, ethToUSD, isEthereumMainnet])
```

Comment:
i think you can use `NativeCurrencyPrice.tsx`

Reply:
I need to pass it as string value for the title as well and this gives me a component which makes it a bit more complex than it is now imo